woocommerce/woocommerce-ios
# Hub Menu: improved element selection and hidden the reviews tab

Part of #5509 

### Description
Some improvements to the Hub Menu:
- Fixed selection of `HubMenuElement`.
- Removed `default` case from `HubMenu`.
- The Reviews tab will be hidden if the hub menu is shown. 
- Updated the unit tests of `MainTabBarControllerTests`.

### Testing instructions
1. Launch the app in debug mode (so the feature flag for the hub menu will be enabled).
2. You should see the hub menu in the tab bar, but the reviews tab should be hidden.
3. The entire area of a menu under the hub menu should be selectable.

### Screenshots
<img src="https://user-images.githubusercontent.com/495617/148065045-bbb82ff8-fd8c-4234-8900-4296213a8032.png" width=300 />


---
